File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> hm=new HashMap<>();
        int idx1=-1,idx2=-1;
        for(int i=0;i<nums.length;i++)
        {
            hm.put(nums[i],i);
        }
        for(int i=0;i<nums.length;i++)
        {
            int val=target-nums[i];
            idx1=i;
            if(hm.containsKey(val)&&idx1!=hm.get(val))
            {
                idx2=hm.get(val);
                break;
            }
        }
        return new int[]{idx1,idx2};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        list_a=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    list_a.extend([i,j])
                    break
        return list_a                    

        